public/app/features/teams/TeamList.tsx

```diff
@@ -89,20 +87,34 @@ export class TeamList extends PureComponent<Props, State> {
     return (
       <tr key={team.id}>
         <td className="width-4 text-center link-td">
-          <a href={teamUrl}>
+          {canReadTeam ? (
+            <a href={teamUrl}>
+              <img className="filter-table__avatar" src={team.avatarUrl} alt="Team avatar" />
+            </a>
+          ) : (
             <img className="filter-table__avatar" src={team.avatarUrl} alt="Team avatar" />
-          </a>
+          )}
         </td>
         <td className="link-td">
-          <a href={teamUrl}>{team.name}</a>
+          {canReadTeam ? <a href={teamUrl}>{team.name}</a> : <div style={{ padding: '0px 8px' }}>{team.name}</div>}
```

Comment:
You'd need to do this via `getStyles` function, and use withTheme2 component wrapper since this is a class component, for ex.: https://github.com/grafana/grafana-enterprise/blob/main/src/public/reports/ReportPage.tsx#L531-L576

Reply:
As discussed with you, let's do that after the release :)